requirements.txt

```diff
@@ -2,3 +2,4 @@ Django==1.10.3
 psycopg2==2.6.2
 pytest==3.0.4
 pytest-django==3.1.2
+django-extensions==1.7.5
```

Comment:
This one is more for development than for production, but I didn't want to spend time right now to have separate settings (I need to check with Two Scoops of Django about what was the best practice to follow in these situation...)